src/main/java/org/EmailCarbonFootprint/GmailAccountDetails.java

```diff
@@ -7,44 +7,45 @@
 public class GmailAccountDetails {
 
 	private String emailId;
-	private String appPassword;
-	
+	private String gmailAppPassword;
+
 	public GmailAccountDetails() {
```

Comment:
This function is already defined below at line number 15, so can we remove it from here?

Reply:
This is not a function, this default constructer.